common/src/test/java/net/adoptopenjdk/icedteaweb/jvm/JvmUtilsTest.java

```diff
@@ -90,4 +96,114 @@ public void testJava8JavaVMArgs() {
             fail(ile.getMessage());
         }
     }
+
+    @Test
+    public void testJava9JavaVMArgs() {
```

Comment:
can you please split this test such that only one call to `JvmUtils.checkVMArgs()` happens in a test method